src/graph/graphcontext.c

```diff
@@ -360,7 +360,6 @@ Index *GraphContext_GetIndex(const GraphContext *gc, const char *label,
 
 	ASSERT(gc != NULL);
 	ASSERT(label != NULL);
-	ASSERT(attribute_id != NULL);
```

Comment:
This assertion is invalid when the function is invoked from `utilize_indices.c`.